app/controllers/finish_installation_controller.rb

```diff
@@ -27,6 +27,7 @@ def register
       @user.password_required!
 
       if @user.save
+        @user.change_trust_level!(1)
```

Comment:
What if the `trust_level` was already > 1?

Reply:
Good catch; I missed the line about loading the existing user if it exists. Fixed now.